wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -72,8 +73,8 @@ export default function ShopModalView(props: ShopModelViewProps) {
           <h3>Offered Diving Trips</h3>
           {props?.isPartnerAccount && (
             <div className={`${style.buttonAddDivingEvents}`}>
-              <Button className="mt-2 btn-lg">
-                Add diving event
+              <Button className="mt-2 btn-lg" onClick={props.openTripCreatorList}>
+                Edit trips
```

Comment:
I don't think you needed to change this, "Add Diving Event" was fine here since adding /editing of them happens inside